system/ThirdParty/Escaper/Escaper.php

```diff
@@ -157,9 +157,21 @@ public function __construct(?string $encoding = null)
         $this->htmlSpecialCharsFlags = ENT_QUOTES | ENT_SUBSTITUTE;
 
         // set matcher callbacks
-        $this->htmlAttrMatcher = [$this, 'htmlAttrMatcher'];
-        $this->jsMatcher       = [$this, 'jsMatcher'];
-        $this->cssMatcher      = [$this, 'cssMatcher'];
+        $this->htmlAttrMatcher =
+            /** @param array<array-key, string> $matches */
+            function (array $matches): string {
+                return $this->htmlAttrMatcher($matches);
+            };
+        $this->jsMatcher       =
+            /** @param array<array-key, string> $matches */
+            function (array $matches): string {
+                return $this->jsMatcher($matches);
+            };
+        $this->cssMatcher      =
+            /** @param array<array-key, string> $matches */
+            function (array $matches): string {
+                return $this->cssMatcher($matches);
+            };
```

Comment:
Sorry. please remove the change.
We don't upgrade Escaper in `develop` branch.

Reply:
Oops, I haven't noticed how that got committed.